api_server/core/views.py

```diff
@@ -11,5 +14,15 @@ class TrialViewSet(viewsets.ModelViewSet):
     ViewSet for Trial model
     """
 
-    queryset = Trial.objects.all()  # pylint: disable=no-member
+    permission_classes = [permissions.IsAuthenticated]
```

Comment:
SO what kind of token is it ? Can you add a curl a example in the readme :)

Reply:
I updated description with examples :)